nucliadb/tests/conftest.py

```diff
@@ -18,6 +18,7 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 pytest_plugins = [
+    "pytest_mock",
```

Comment:
Can we remove the `conftest.py` in the `writer` directory now?

Reply:
Yeah, I'll change to a more explicit approach in conftest and avoid this hack

Reply:
It is deleted. This is a general conftest for all migrated packages (currently reader and writer)

Reply:
Uhm, I don't see it deleted in this PR. :confused: